start_notebook.sh

```diff
@@ -1,19 +1,23 @@
 #!/bin/bash
-
+echo "Note: This script will startup pyspark with jupyter nootbook"
 # Check environment variables
 if [ -z "${BIGDL_HOME}" ]; then
     echo "Please set BIGDL_HOME environment variable"
+    echo "For example:"
```

Comment:
the above line is sufficient enough, every programmer should know what does set environment variable mean . i don't think we need to add extra info

Reply:
When user install bigdl from source, it's known where is the BIGDL_HOME.
But when install bigdl by PIP, it's hard to know by normal user. 
There is no document to tell user where is the BIGDL_HOME.
I have such problem and I find the answer after check.
Here, I want to give some clue to user for the case by PIP installation.

Reply:
it's irrelevant, and also it's not about setting the env,  it's about how to find you bigdl_home path.

Reply:
Yes.
In the script help print, I want to guide user for the question: how to find you bigdl_home path.
I don't know where is good place to hint user.
So I put it in this script.

Do you think where is better?
I could update this info in right place.

Reply:
different installations have different results, also users may build it from scratch. I would suggest we don't add it, the promoted message intends to remind user to set variable, that's all

Reply:
OK, I will remove them.